src/hooks/useWalletAsset.ts

```diff
@@ -76,12 +76,12 @@ const useWalletAsset = () => {
             wallet.scheme.miniscriptScheme?.usedMiniscriptTypes.includes(
               MiniscriptTypes.INHERITANCE
             ) && { tag: 'Inheritance Key', color: Colors.Aquamarine },
-            { tag: getSchemeTag(wallet as Vault), color: Colors.LightMossGreen },
+            { tag: getSchemeTag(wallet as Vault), color: Colors.DesertSand },
           ].filter(Boolean);
         default:
           return [
-            { tag: 'Multi-key', color: Colors.mintGreen },
-            { tag: getSchemeTag(wallet as Vault), color: Colors.LightMossGreen },
+            { tag: 'Multi-key', color: Colors.LabelLight3 },
```

Comment:
It's not enough to assign the colors like this, the colors need to be consistent, ie first pill color is the same for all then second is also same for all etc,